.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
+*.log
```

check your `log4j2.xml`
```suggestion
logs/*
```